### ISSUE TITLE
Organizacion de componentes

### ISSUE THREAD

**Issue by martinbavio** [daianaszwimer/wwcba-workshop-solution#1]
Me pregunto si organizarlo por _domain_ (home, blog, etc) como estas haciendo aca, tiene sentido para un proyecto chico y que esta apuntado a que aprendan principiantes.

En lo personal, para un proyecto de esta envergadura, usaria una carpeta de componentes, otra de hooks, quizas una de lib con cosas extra, y ya.

**Comment by daianaszwimer** [comment]
Claro, lo hice porque elegí seguir TAO Of React https://alexkondov.com/tao-of-react/ que lo organiza así. Me gusta mucho organizar de esa manera jaja pero puedo agregar esa aclaración

**Comment by martinbavio** [comment]
@daianaszwimer esta piola, pero las recetas no siempre aplican a todos los contextos. Yo tambien empece usando React buscando un _holy grail_ con respecto a la organizacion de los componentes, y de a poco me fui dando cuenta que no es tan importante como parece. 

Lo mas importante, desde mi humilde opinion, es que una persona nueva a React entienda que uno puede crear la organizacion que quiera y ponerle los nombres que se le ocurra, pero al fin y al cabo, no es mas que un compendio de componentes, y que justamente el concepto de componente es el verdadero _separation of concerns_ que propone React.

**Comment by daianaszwimer** [comment]
@marbiano dale, me gusta que el foco no se vaya de lo importante así que voy por lo que me decís. Entonces meto todo en componentes, no? Home, Blog y esos también, todos en la bolsa de componentes
(Me voy a dormir así que respondo mañana si llego) (muchas gracias)

**Comment by martinbavio** [comment]
Sep, yo haria una carpeta `components` y ahi meteria todos los componentes. Y explicaria en la charla que eso puede organizarse de otras maneras, que a React realmente no le interesa eso.
